### PR TITLE
Add WORKSPACE_DIR handling for compose commands

### PR DESCRIPTION
**Pull request by ryansch**
Also extracts `RunIt`

### CODE REVIEW

File: bin/dev [hunk 1]
```diff
@@ -128,16 +131,19 @@ HEREDOC
...
-        stdout: $stdout,
-        stderr: $stderr
-      )
-
-      puts
+      RunIt
+        .new(log_level: Logger::INFO)
+        .subprocess(
+          cmd: args,
+          opts: {
```
Comment (ryansch) [inline bin/dev:140]: I'm tempted to get rid of `opts` and use `**opts`.

**Review by morganick — APPROVED** [review]
Nice job sir.

**Comment by ryansch** [comment]
@morganick I cleaned up the subprocess call signature.

**Review by morganick — APPROVED** [review]
I like it

**Pull request merged**